### ISSUE TITLE
[4.x] [5.x] MFA Backup Codes [a11y] [UI]

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#43449]
When you enable any of the MFA methods it is **essential** that you also print out and keep a set of backup codes.

### Steps to reproduce the issue
Enable any of the MFA methods and then try to print out the backup codes

### Joomla 4
If you have good eyesight that can see the subtle colour change you can find the link
![image](https://github.com/joomla/joomla-cms/assets/1296369/fee96c20-bbac-4c87-b860-e58e4e25aa70)

### Joomla 5
There is no visible indicator for the link. It is only because I knew that there should be one and I viewed the source that I was able to find it.

![image](https://github.com/joomla/joomla-cms/assets/1296369/206bac28-b499-488c-8b1c-72120bbd1752)

**Comment by brianteeman** [comment]
Updated original post with correct screenshots

**Comment by brianteeman** [comment]
See https://github.com/joomla/joomla-cms/pull/43450
